juju/charm.py

```diff
@@ -1,62 +1,69 @@
 from juju.framework import Object, Event, EventBase, EventsBase
 
 
-class InstallEvent(EventBase):
+class JujuEvent(EventBase):
     pass
 
-class StartEvent(EventBase):
+class JujuHookEvent(EventBase):
```

Comment:
I'd call this a HookEvent for the same reason. We don't say JujuInstallEvent below, for example.